Jenkinsfile

```diff
@@ -76,8 +81,9 @@ pipeline {
           if (currentBuild.getBuildCauses('jenkins.branch.BranchIndexingCause') &&
               BRANCH_NAME == "develop") {
               print "INFO: Branch Indexing, aborting job."
-              currentBuild.result = 'ABORTED'
-              error('Stopping early')
+              currentBuild.getRawBuild().getExecutor().interrupt(Result.SUCCESS)
+              sleep(3)   // Interrupt is not blocking and does not take effect immediately.
```

Comment:
The magic sleep ...  the glue of the Internet. How safe is 3 (I assume seconds)? The whole pipeline takes long time, so we can also do 10s ;)

Reply:
yea :-) The example I took this from had 1 second so I thought that 3 should be enough, but who knows :-)